tests/Skipper/Skipper/config/single_skipped_rule_config.php

```diff
@@ -6,11 +6,6 @@
 use Rector\Config\RectorConfig;
 use Rector\DeadCode\Rector\ClassMethod\RemoveUnusedPromotedPropertyRector;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->rules([
-        InlineConstructorDefaultToPropertyRector::class,
-        RemoveUnusedPromotedPropertyRector::class,
-    ]);
-
-    $rectorConfig->skip([InlineConstructorDefaultToPropertyRector::class]);
-};
+return RectorConfig::configure()->withSkip([InlineConstructorDefaultToPropertyRector::class])->withRules(
+    [InlineConstructorDefaultToPropertyRector::class, RemoveUnusedPromotedPropertyRector::class]
+);
```

Comment:
These should be indented with single call per line.

Reply:
any ecs config can be applied?

Reply:
I updated to use new line per-call